SwanShare/swanshare/serverextension.py

```diff
@@ -1,9 +1,8 @@
-from notebook.base.handlers import APIHandler
-from notebook.utils import url_path_join
+from jupyter_server.base.handlers import APIHandler
```

Comment:
I think it would help to add two - three lines to the commit message on why we are moving from notebook to jupyter_server.